scanner/src/routes/scan/+page.svelte

```diff
@@ -85,24 +101,35 @@
   })
 </script>
 
-<div class="relative barcode-scanner-area-wrapper">
-  <div class="relative barcode-scanner-area">
-    <img alt="subtract" class="subtract-image" src={Subtract} />
-    <div class="square surround-cover" />
+{#if isScanning}
+  <div class="relative barcode-scanner-area-wrapper">
+    <div class="relative barcode-scanner-area">
+      <img alt="subtract" class="subtract-image" src={Subtract} />
+      <div class="square surround-cover" />
+    </div>
+    <div class="relative">
+      <p class="description">Scan a Product Passport QR code to view product data</p>
+    </div>
   </div>
-  <div class="relative">
-    <p class="description">Scan a Product Passport QR code to view product data</p>
+  <div class="relative example-code-wrapper">
+    <div class="example-description">
+      Here's an example to identify an <strong>IOXIO Tag</strong>
+    </div>
+    <div class="example-code">
+      <img alt="An example IOXIO Tag" src={IoxioTagExample} />
+    </div>
   </div>
-</div>
-<div class="relative example-code-wrapper">
-  <div class="example-description">
-    Here's an example to identify an <strong>IOXIO Tag</strong>
+  <Documentation />
+{:else}
+  <div class="relative failed-verification-wrapper">
+    <img src={Failed} alt="" aria-hidden="true" />
+    <p>Code failed authenticity verification<br />Continue anyway?</p>
```

Comment:
```suggestion
    <p>Code failed authenticity verification</p>
    <p>Continue anyway?</p>
```
`<br/>` is evil.

Reply:
Not committing because I can't confirm it looks correct afterwards, so please test and adjust as necessary.